### ISSUE TITLE
online.js + docker

### ISSUE THREAD

**Issue by alexunderboots** [immisterio/Lampac#8]
online.js не выдает результатов при запуске в докере
остальные плагины работают нормально

**Comment by alexunderboots** [comment]
крутится колесико и всё на этом

**Comment by tsynik** [comment]
> крутится колесико и всё на этом
и смысл крутить lampac в докере когда он и без него отлично работает?

**Comment by krilzov** [comment]
Приветствую друзья! У меня аналогичная ситуация, при использовании lampa через реверс прокси (с использованием сертификата по https без указания порта 9118) плагин online.js выдаёт: Ошибка сервер не отвечает на запрос. Остальные плагины работают исправно. По локальному ip:9118 всё также работает. Есть решение проблемы?

**Comment by beshkenadze** [comment]
> Приветствую друзья! У меня аналогичная ситуация, при использовании lampa (установлена на ubuntu) через реверс прокси (с использованием сертификата по https без указания порта 9118) плагин online.js выдаёт: Ошибка сервер не отвечает на запрос. Остальные плагины работают исправно. По локальному ip:9118 всё работает идеально. Есть решение проблемы? p.s. при замене кода online.js на код плагина онлайн мод - всё работает

Откройте консоль разработчика и посмотрите ошибки. Скорее всего пробует подгрузить по `http` со страницы `https`.
Если это так, то надо удалить эти плагины и добавить вручную через `https`.

**Comment by alparo** [comment]
Если отключить Force SSL в настройках nginx для Lampac, то родной online плагин начинает работать. Но тогда почему-то все постеры начинают очень медленно загружаться. Пока не нашел решения.

@beshkenadze подскажите, пожалуйста, поподробнее процесс диагностики через консоль разработчика. Имеется в виду инструменты разработчика в браузере?

**Comment by beshkenadze** [comment]
@alparo да, браузерный. Force SSL само название говорит, что принудительно заворачивает в https все запросы. Если вы в России, то скорость может быть низкая из-за РКН и их оборудования. 
Попробуйте не использовать https в целом и скорость должна быть норм.

**Comment by toshk0** [comment]
Мне помогло в init.conf добавить:
`"listenscheme": "https"`

После этого всё заработало с обратным прокси и https
